### PR TITLE
Adding Contribute / Bug Link

### PR DESCRIPTION
**Pull request by Pyrodius0**
.env needs the following values:
LINK_GITHUB=https://github.com/thatsmybis/thatsmybis
LINK_GITHUB_BUG=https://github.com/thatsmybis/thatsmybis/issues/new/choose

### CODE REVIEW

**Comment by Lemmings19** [comment]
You'll want to update `.env.example` with these values.

**Comment by Lemmings19** [comment]
I think there's only room for one, and I don't want to crowd too much onto this menu. (I've avoided adding a great many things to keep it easily digestible)

Let's just remove the donate button. We'll find somewhere else for it to live. Maybe the footer.

image

File: resources/views/layouts/nav.blade.php [hunk 1]
```diff
@@ -157,6 +157,14 @@
                 @endif
             @endif
         </ul>
+        <ul class="navbar-nav">
+            <li class="nav-item mr-3">
+                <a href="{{ env('LINK_GITHUB') }}" target="_blank" class="nav-link active font-weight-bold">
+                    <span class="fab fa-github"></span>
+                    Contribute
+                </a>
+            </li>
```
[the remark this answers is not in the document]
Comment (Pyrodius0) [reply resources/views/layouts/nav.blade.php:166]: Good call out

**Comment by Pyrodius0** [comment]
I know this can look weird.. but I left it as is because it makes it stand out. 

image

**Comment by Lemmings19** [comment]
`.env.example` needs these new fields added. After that, it should be good for a merge.

**Pull request merged**